### ISSUE TITLE
Probably check eligibility when someone signs up for a crew

### ISSUE THREAD

**Issue by MasterOdin** [rpiambulance/website#59]
Right now, the check is only done for showing or not showing the button, but doesn't stop someone from just  sending a direct request to the API to bypass the button to sign up for any position regardless of eligibility.

**Comment by ddbruce** [comment]
@lramos15 was this fixed in #104?

**Comment by lramos15** [comment]
No. This is about running the eligibility check when requesting to signup rather than to see if we should put the signup button there or not
